runtime/hydradx/src/lib.rs

```diff
@@ -1096,7 +1036,9 @@ construct_runtime!(
 		OmnipoolLiquidityMining: pallet_omnipool_liquidity_mining = 63,
 		OTC: pallet_otc = 64,
 		CircuitBreaker: pallet_circuit_breaker = 65,
+		// DCA = 66
```

Comment:
It is not removed. It was placed further down because of ordering. I thought it made sense to add a comment here where DCA "should be" in the order if we were ordering by index.